docker/lumino-explorer/assets/startExplorer

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+source "${HOME}/.sdkman/bin/sdkman-init.sh"
+
+INIT_LOG=/root/init.log
+EXPLORER_LOGS_FOLDER="/var/logs/explorer"
+EXPLORER_ROOT_FOLDER="${HOME}/lumino-explorer/lumino-explorer-api"
+EXPLORER_POM_FILE="${EXPLORER_ROOT_FOLDER}/pom.xml"
+EXPLORER_DATABASE_SETUP_FILE="${EXPLORER_ROOT_FOLDER}/src/main/resources/database/lumino-explorer-api-database-setup.js"
+
+if [ "${RSK_ENDPOINT}" == "" ]; then
+    RSK_ENDPOINT="http://rsk-node:4444"
+fi
+
+if [ "${EXPLORER_PORT}" == "" ]; then
+    EXPLORER_PORT=8080
+fi
+
+if [ "${TOKEN_NETWORK_REGISTRY}" == "" ]; then
+    TOKEN_NETWORK_REGISTRY=0xed8c9163F888Bc2f9C4F299325003DA5fC8676DD
```

Comment:
im at least confused about this hardcoded address but maybe when you give us a mini workshop we can improve the documentation to run the docker container and fix those kind of things

Reply:
Basically all the default values are regtest network values.